### PR TITLE
Wallet API Updates, CLI Review

### PR DESCRIPTION
**Pull request by martindale**
Want to merge these API changes into `feature/node-16` — lots of dependency cleanup, first security pass, improved CLI.

**Notable Changes:**
- All critical security issues resolved
- All dependencies pinned with strict versioning
- Re-worked "Environment Setup" (`@fabric/core/contracts/setup`)
- Improved `Key` and `Wallet` classes
- ZMQ Service now included by default (`@fabric/core/services/zmq`)

**Sign-offs:**
- [x] @anandsuresh for `fabric setup` and `fabric` environment setup semantics

### CODE REVIEW

**Comment by martindale** [comment]
@anandsuresh must faster build here — were you using the `semistandard` or `semistandard-loader` packages?

**Comment by martindale** [comment]
All tests passed \o/

File: book.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "title": "Fabric Developer's Handbook",
   "description": "Build & Run P2P Applications with Fabric",
   "language": "en",
-  "plugins": ["edit-link"],
```
Comment (martindale) [inline book.json:6]: Related to #6

**Comment by martindale** [comment]
I'll resolve the coverage diff in future work — `feature/node-16` remains an objective improvement over `master`

**Comment by martindale** [comment]
@terryisterry https://github.com/FabricLabs/fabric/pull/93/files#diff-4f74b24c8cd7466c1c46dc972d8e284ed47091160afda95a432cc000f2eceb4aR41-R42 for community

**Review by anandsuresh — APPROVED** [review]
Looks good to me. A few nit-picks that can be converted into issues/tickets for future PRs.

**Pull request merged**